backends/tfhe-cuda-backend/cuda/src/utils/helper_multi_gpu.cuh

```diff
@@ -18,9 +19,10 @@ void multi_gpu_alloc_array_async(cudaStream_t *streams, uint32_t *gpu_indexes,
 }
 /// Copy an array residing on one GPU to all active gpus
 template <typename Torus>
-void multi_gpu_copy_array_async(cudaStream_t *streams, uint32_t *gpu_indexes,
-                                uint32_t gpu_count, std::vector<Torus *> &dest,
-                                Torus *src, uint32_t elements_per_gpu) {
+void multi_gpu_copy_array_async(cudaStream_t const *streams,
+                                uint32_t const *gpu_indexes, uint32_t gpu_count,
+                                std::vector<Torus *> &dest, Torus *src,
```

Comment:
Shouldn't "src" be `const`?

Reply:
Yes sure! I made const all the things absolutely necessary to be able to build & generate the correct bindings, but not more, there might still be places where some variables should be passed const and are not.